modis preprocessing fails due to thrift packing issue

`hadoop jar forma-0.2.0-SNAPSHOT-standalone.jar forma.hadoop.jobs.modis "s3n://modisfiles/MOD13A1/" "s3n://pailbucket/blzmasterpail/" "{2000,2001,2002,2003,2004}*" :BLZ`

``` text
Caused by: java.lang.RuntimeException: java.lang.IllegalArgumentException: No implementation of method: :pack of protocol: #'forma.thrift/IPackable found for class: forma.schema.IntArray
    at cascalog.ClojureCascadingBase.applyFunction(ClojureCascadingBase.java:71)
    at cascalog.ClojureMap.operate(ClojureMap.java:34)
    at cascading.flow.stream.FunctionEachStage.receive(FunctionEachStage.java:86)
    ... 97 more
Caused by: java.lang.IllegalArgumentException: No implementation of method: :pack of protocol: #'forma.thrift/IPackable found for class: forma.schema.IntArray
    at clojure.core$_cache_protocol_fn.invoke(core_deftype.clj:527)
    at forma.thrift$eval8067$fn__8068$G__8058__8073.invoke(thrift.clj:104)
    at clojure.lang.Var.invoke(Var.java:415)
    at clojure.lang.AFn.applyToHelper(AFn.java:161)
    at clojure.lang.Var.applyTo(Var.java:532)
    at cascalog.ClojureCascadingBase.applyFunction(ClojureCascadingBase.java:68)
    ... 99 more
```

--- a/src/clj/forma/source/hdf.clj
+++ b/src/clj/forma/source/hdf.clj
@@ -184,7 +184,7 @@ as a 1-tuple."
         ret (int-array (* width height))]
     (.ReadRaster band 0 0 width height ret)
     (map-indexed (fn [idx xs]
-                   [idx xs])
+                   [idx (vec xs)])
                  (partition chunk-size ret))))
 
 ;; ### Metadata Parsing

